python/pyspark/sql/session.py

```diff
@@ -670,9 +670,7 @@ def _create_shell_session() -> "SparkSession":
             conf = SparkConf()
             assert SparkContext._jvm is not None
```

Comment:
MyPy can understand `_jvm` assert so it doesn't require `# type: ignore[attr-defined]`.